starter-code/src/app/contact-list/contact-list.component.html

```diff
@@ -1,18 +1,30 @@
-<div>
-  <label for="contact-name"></label>
-  <input type="text" [(ngModel)]="newContact.name">
-</div>
-<div>
-  <label for="contact-email"></label>
-  <input type="text" [(ngModel)]="newContact.email">
-</div>
-<div>
-  <label for="contact-phone"></label>
-  <input type="text" [(ngModel)]="newContact.phoneNumber">
-</div>
-<div>
-  <label for="contact-image"></label>
-  <input type="text" [(ngModel)]="newContact.image">
-</div>
+<ul class="row">
+  <li *ngFor="let oneContact of contacts" class="col-md-4">
+    <h4>Name: {{oneContact.name}}</h4>
+    <p>Email: {{oneContact.email}}</p>
+    <p>Phone number: {{oneContact.phoneNumber}}</p>
+    <img src='{{oneContact.image}}' />
+    <br />
```

Comment:
Careful with using `<br>` to align your HTML. It it usually better to use class styling, flexbox, etc.